README.md

```diff
@@ -0,0 +1,17 @@
+# browserify-rsa
```

Comment:
the tendency is to make the special files allcaps, but they are technically case insensitive due to windows file system being case insensitive and most mac file systems being mostly case insensitive.